wiki/Home.md

```diff
@@ -0,0 +1,26 @@
+
+Welcome to the unified-codes wiki!
+
+"unified-codes" is intended to complately replace the original universal drug codes server. 
+* The "original" universal drug codes server is currently located at https://sussol.github.io/universal-drug-codes-server/.
+* "unified-codes" (aka V2) is not yet deployed.
+
+## System Overview/Design [TO COMPLETE]
+<!-- This is quite high level - A diagram may be useful -->
+* Database 
+  * [Dgraph](https://dgraph.io/docs/)
+* APIs 
```

Comment:
As above, can just go with `Legacy API` here, I think.

Reply:
Sorry, I thought that we had a diagram to hand which we could drop in here. But no. It's a good idea though & we'll need to knock something up, maybe next week.

Reply:
@wlthomson Yep - makes sense if GraphQL/DB are bundled as per above
@mark-prins Yeah - I had an initial go with https://mermaid-js.github.io/mermaid/#/ as it was in the stackedit template but it was embarrassingly hard to read.